docs/current_docs/integrations/_integration.mdx.template

```diff
@@ -2,16 +2,16 @@
 slug: /integrations/<name-of-integration>
 ---
 
-# <Name of Integration>
+# <name of integration>
```

Comment:
I wouldn't include an update to the template in this PR. Might be in by mistake?